.gitignore

```diff
@@ -20,4 +20,5 @@ result
 test_cert
 
 .vscode
-.my-shell.nix
\ No newline at end of file
+.my-shell.nix
```

Comment:
i'm interested in learning how other devs set up their dev environment.
how about we actually track dev-specific shells in git? could name it `.<github-handle>-shell.nix`

Reply:
This is not holochain specific. It's just a work around to make the nix environment selector work for vscode

Reply:
it works here for me if i point it to the _shell.nix_ file. the _default.nix_ won't work.

Reply:
Yeh but I don't want to use that shell because it's got an older version of rustc and it's missing tools that I use.

Reply:
so it _is_ specific to your holochain development needs. that's why i'm interested in tracking it in the repo as well.